packages/components/pie-modal/src/constants.ts

```diff
@@ -0,0 +1,8 @@
+// MODAL EVENTS
+
+/**
+ * Event name for when the modal is closed.
+ *
+ * @constant
+ */
+export const ON_MODAL_CLOSE_EVENT = 'pie-modal-close';
```

Comment:
can't we just add this to defs.ts?

Reply:
I'm not really sure tbh, but I would assume defs stand for definitions :D 
sorry, i didn't use the defs convention before but I would just use it to not have too many files unless its against the convention

Reply:
This is what I read so i am not sure it is the same? Happy to move it there but maybe we need some clarity on this file and the purpose it can be used with the correct intention?

`TypeScript definition files (.d.ts files) are used to provide type information for external JavaScript libraries or modules that do not have built-in TypeScript support. These definition files describe the shape and types of the existing JavaScript code, enabling TypeScript to perform static type checking and provide enhanced tooling support.`

So to me it looks like the defs file is for defining the shape of javascript so TS can allow for type checking and provide better support.

Reply:
aha then, in this case, we aren't following that as we export things like ButtonSizes, headingLevels, etc which are kinda a constant to some extent as well.

Reply:
Doesn't need to be done as part of this PR, but would be good to get some advice on this from @thejfreitas or @davidpn11 to check that we're defining these in the right way (if there is a 'right' way 😄